### DISCUSSION TITLE
Hi Sage Community! Long time (past) user! Quick Question - Why are people still using SageTV?

### DISCUSSION THREAD

**Discussion by joejanos** [OpenSageTV/Community#104] [Q&A]
I've been a long time user of SageTV! I miss the comradery of this community! But I've switched to plex and it seems a lot easier to setup and use. Are there any new instances of sageTV being created? Why would you choose to install sageTV as opposed to Plex?

Not casting stones. Just curious!

**Comment by nyplayer** [comment]
Plex DVR is the worst there is it is so basic No Actor Favorites ... Keyword Favs etc ... I prefer Channels DVR and NEXTPVR ... SageTV fanarrt etc is getting long in the tooth.  To View Media I use EMBY not Plex.

**Reply by joejanos** [reply]
Thanks! I just installed Channels as well, but I'm not sure about the monthly fee associated with the DVR service.

**Reply by Zogg44** [reply]
You get what you pay for. $8 per month for supported software that you use probably every day is plenty worth it.

**Reply by nyplayer** [reply]
Do not forget that Price Includes guide data and full support and all updates.

**Reply by joejanos** [reply]
I am using the version of Channels connected to my HDHomerun. This is free and gives channel guide. But, no DVR functions.

**Reply by LoisGNS** [reply]
I am under the impression that Channels works only with HDHomeun (plus, maybe, other network tuners?). I would consider it as a fallback if needed if it could work with my Hauppuage tuners, of which I have many. Does anyone have a resource or knowledge regarding supported tuners?

**Reply by nyplayer** [reply]
HdHomerun tuners no Hauppauge support. If you absolutely need other tuner support look at NEXTPVR.

**Reply by Jaroch76** [reply]
Because I like SageTV.  It still works extremely well for playing TV and videos, is reliable, highly customizable, and integrates nicely with my old-school remote control.

**Comment by wayner9** [comment]
I still use SageTV as my primary media playback.  I have used Plex intermittently for the last decade or so, and I just installed Jellyfin.  They all use the same media library.

For playback of existing video files Plex or Jellyfin are better in some ways.

But for recording TV, SageTV is still better, and the same for playback of recorded content.  Most of what I record is sports and SageTV is great at recording and skipping forward.  I can instantly skip forward 10 seconds, or 2 minutes for commercials.  There is no transcoding, just direct playback.  Plus SageTV has a ton of useful plugins, like Sage Recording Extender, etc.  And you can easily playback content that is still recording, which is an issue with other software.  And they generally don't like sports because it doesn't have real metadata.

**Comment by Thomptronics** [comment]
Because it still works!

Oh, and after many years for platforms like Plex to catch up, Sage TV _still_ has the best commercial skip features.

**Reply by joejanos** [reply]
I don't know about that! Plex has a commercial remove or mark for skip. The older shows, it works really well! The newer shows sometimes, I have to turn it off.

**Reply by Thomptronics** [reply]
Plex is "OK" at commercial skip, but it can't do it in real time. For example, I record the nightly news but don't start watching it until it's been on five or ten minutes. This way, Sage TV automatically skips commercials for me even though the show hasn't even ended.

Also, last I checked (and this could have been "fixed" by now) Plex literally _deletes_ what it thinks are commercials. So, it if gets it wrong, you don't get all of your show.

**Reply by joejanos** [reply]
You are correct about the real-time commercial skipping! That is definitely a strength. As for deleting the commercials in Plex, it allows it to be done two ways, either deleting (where the footage is gone), or marking, where it does it similar to SageTV, except you have to hit the skip button at each commercial time. SageTV does it better. But, I'm getting used to the Plex com skip. Hopefully it gets better!

**Reply by Thomptronics** [reply]
Glad to hear Plex added the commercial-marking option. And, don't get me wrong -- I like Plex.  I actually bought a lifetime subscription and use it frequently to stream my music library to my Alexa devices. It is also my current backup plan for the sad day my Sage TV system kicks the bucket.

**Reply by LoisGNS** [reply]
I just use the skip forward & backward buttons, rather than marking commercials for skipping. It's only slightly more hands-on and I don't have to worry about marking errors. (Not that I've ever tried the comskip plugin, but it struck me as too much trouble).

**Comment by TGsell1** [comment]
My Sage system provides everything I need and I don’t really suffer from the lack of development since my server and devices are ‘frozen’.  I am more worried about changes by my cable company and cable card.  My system plays back in network via HD300s or FireSticks almost perfectly (no cc on fire stick) For remote play we use AirVideo HD perfectly and enables downloading to apple devices.  I have 4OTA tuners and three CableCard tuners via Windows Media Center miniPCs and network encoders.  All updates ‘frozen’ so the system just works the same day in day out.   Sent from my iPhoneOn Dec 28, 2023, at 11:11 AM, Thomptronics ***@***.***> wrote:﻿
Because it still works!
Oh, and after many years for platforms like Plex to catch up, Sage TV still has the best commercial skip features.

**Reply by joejanos** [reply]
Thank you for this! It really sounds like you have drank the SageTV Koolaid! I was like that until my capture cards were starting to go bad. I then transitioned everything over to Plex. I have Jellyfin installed as well, but that doesn't seem to be ready for primetime! I am now experimenting with Channels as a live TV presentation for my wife that also misses SageTV.

**Comment by trk2** [comment]
Nothing is faster at skipping than Sage and the Plex UI is awful.

**Reply by nyplayer** [reply]
I personally prefer Channels DVR it has Comskipping as good as SageTV and has an added feature that SageTV does not have the ability to Edit Comskip cut points. Also the ability to add your own M3U as tuners. I have finally shutdown all my SageTV Servers including my test System and went Strictly Channels DVR and EMBY. I left SageTV finally because it is no longer a supported project. Relying on other users is great but how long do you think that will last ?

The Ability to record Pluto TV LiveTV is an added bonus.

Comskip Editor
https://i.postimg.cc/wTd11cbG/Screenshot-2023-12-28-145917.png

**Reply by Zogg44** [reply]
I've also retired my Sagetv server and use Channels for recording/watching TV,  and Plex for anything I download. Channels handles commercial skipping extremely well, and also closed captions. Sage seemed very picky about things I download, and often refused to play subtitles, whereas Plex plays almost everything with no problems.

**Reply by Thomptronics** [reply]
I played with Channels DVR a while back and can't remember why I rejected it (cost?).  But, I am curious: Does Channels DVR support Comskipping of live shows ala what Sage TV does, or do you have to wait for the show to end so it can then process the recording?

**Reply by nyplayer** [reply]
There is an Option to start Comskip when you start watching the show... SO if I start watching a recording half hour into it Comskip will start ... Comskip is fast so you will get markers and automatically skip them.

"Start live detection when watching in-progress recording"

**Reply by joejanos** [reply]
Thanks for the info on Channels! It might be worth the $8 per month. I didn't know it can record Pluto!!! That is a game-changer!

**Reply by wayner9** [reply]
But Plex Channels does give you the main cable sports channels like ESPN, or TSN here in Canada, does it?  It gives you a bunch of Live TV channels, but not everything.
And does Plex support tuners like the HD-PVR or other HD encoders other than OTA?

**Reply by emveepee** [reply]
Comskip works fine in NextPVR, always has and it works on in-progress with UI clients.    Ultimately it is comskip and the edl file that do the bulk of the work not the PVR program.  NextPVR also supports SRE without the need for a local server. ChannelsDVR is primarily for Americans so no interest here.

As for why SageTV, it is no doubt to me that it is the HD300 users keeping it alive, people went on a binge buying them.I wish Narflex let me create a NextPVR for it when I asked, I knew the Sigma SDK it uses.  They do work, but let's see what happens with ATSC 3.0.

**Reply by joejanos** [reply]
I know that channels is ATSC 3.0 compatible. I have some stations that are coming over using that protocol. Plex cannot read them and ignores those stations.

**Comment by nyplayer** [comment]
@joejanos 

They do have a yearly subscription for $80 and they also have a Trial Period. Forgot to mention streaming out of home is fantastic.

**Comment by TGsell1** [comment]
If you get some time to work with Comskip it’ll be worth it.   Works like 95% of the time.  Sent from my iPhoneOn Jan 1, 2024, at 9:28 PM, LoisGNS ***@***.***> wrote:﻿
I just use the skip forward & backward buttons, rather than marking commercials for skipping. It's only slightly more hands-on and I don't have to worry about marking errors. (Not that I've ever tried the comskip plugin, but it struck me as too much trouble).

**Comment by sic0048** [comment]
I mean why switch if what I have works perfectly fine?  I started using SageTV in 2007.  After all these years it is still one of the best DVR systems out there (although I realize other options have arguably caught up to it).    I use a cable card tuner and when that system stops working or I change service providers, I'll look into alternatives.  Until then I'm sticking with SageTV.

**Reply by wayner9** [reply]
I switched to SageTV from Windows MCE around 16 years ago.  The impetus was that MCE did not support the HD-PVR so I could not record cable HD channels.  The one slightly annoying thing to me is that SageTV doesn't really support 4K, I watch a lot of sports and some of it is now in 4K.  But then where would I play it back as there aren't 4K extenders and I have found the AndroidTV client to be buggy when I have used it.  And that is the other slightly annoying thing with SageTV - you can't play 4K or H.265 content, at least not on extenders.  But I am willing to live with those drawbacks for now.

**Comment by EricDePouleG** [comment]
Man I tried to get the latest something other then SageTV.  Kodi on FireStick and NextPVR.  Could not understand the channel numbering sent to kodi, sequential numbering?  And with OTA and DirecTV I've got 1500+ channel, luv holding the down button on the wheel to get to a channel, no number entry?  Custom Groups of channels didn't flow to kodi.   Setting up recurring records was no fun, finding future airing, again no fun.   Did like the fanart for my movies on my nas.   But live TV stuff.  SAGETV hands down...   But maybe I'm old and nobody watches OTA or DTV anymore...   I'm gonna stay with SageTV as long as I can.   And of course the Wife Factor with sage... Priceless..

**Reply by emveepee** [reply]
NextPVR has it own clients too for Android/FireTV, Apple, Roku, LGOS, Samsung Tizen, macOS and Windows since many users don't like Kodi and prefer a simpler 10' interface.  I personally use the web client for searching and scheduling because it is much more efficient with a mouse and keyboard.

Kodi PVR defaults to using their own channel number, but you can use backend numbering if you change PVR setting  You can enter a channel number with  a PVR-type remote or keyboard, major minor are assumed if there is no match.   Channel groups from NextPVR do go to Kodi too, not sure what is wrong with your setup.  For a better comparison how does the SageTV interface to Kodi work?

**Reply by EricDePouleG** [reply]
i may give nextpvr and kodi another try, though (OTA and Dtv will stay with sagetv at this time).  I'll try splitting OTA and run nextpvr on a separate machine.   I also read that backend channel numbering could be passed to kodi.  but the killer is the scroll or search by number instead of just typing 3 digits to get to that channel in the guide.   I'm also a fan of hd300, turn it on and it finds the sagetv server and you're in.   No firestick, select kodi, step (and be careful hitting the back arrow on the firestick remote or you return to the firestick (ie leaving kodi).   There's no kodi to sage, no need, sage is all in one (server records and deals with the user ui on a remote box).  Agree with TGsell1 below, lack of development or flashing ui or missing 5.1 etc, but isn't a miss for me.   Don't miss loss of 5.1 on tv.  UI is easy (yes I'm used to it.)  And lack of dev, well not sure what I'm missing that needs to be added to sage.    I'm sure I'll keep looking, but there's no fire at the moment forcing to.   Thank you sagetv people were ever you might be.

**Comment by TGsell1** [comment]
Good to hear that! Same with me.  Lots of Sage users complain about a lack of development and support however my server and associated systems are ‘frozen’ (no updates) and can be reinstalled easily with disc image archives if something goes wrong.  One legit complaint is lack of advanced audio eg 5.1 but honestly for the majority of tv my wife and I watch it’s not missed.  Lack of closed captions using Firestick etc is a minor issue for which I just watch in an HD300.  Bottom line; I have yet to hear anyone in this forum suggest a better overall solution.   Sent from my iPhoneOn Jan 17, 2024, at 10:27 PM, EricDePouleG ***@***.***> wrote:﻿
Man I tried to get the latest something other then SageTV.  Kodi on FireStick and NextPVR.  Could not understand the channel numbering sent to kodi, sequential numbering?  And with OTA and DirecTV I've got 1500+ channel, luv holding the down button on the wheel to get to a channel, no number entry?  Custom Groups of channels didn't flow to kodi.   Setting up recurring records was no fun, finding future airing, again no fun.   Did like the fanart for my movies on my nas.   But live TV stuff.  SAGETV hands down...   But maybe I'm old and nobody watches OTA or DTV anymore...   I'm gonna stay with SageTV as long as I can.   And of course the Wife Factor with sage... Priceless..

**Comment by willkramer** [comment]
I sadly no longer use SageTV - I switched over to Apple TV for streaming and use [Infuse Pro](https://apps.apple.com/us/app/infuse-video-player/id1136220934) ($9.99/yr) for accessing my movie library. I held out for as long as possible before finally retiring the server and my 6 or 7 HD300s/200s a couple years ago. Before moving on, I even wrote workarounds so Sage could control a Roku streaming stick as if it was a cable box so I could access streaming services - however, the lack of guide data made that a limited benefit effort.

Sage was brilliantly designed. Sure, its UI wasn't as flashy as today's streaming apps, but in terms of efficiency and thoughtfulness? Unmatched. Switching from Sage was like moving from a super intuitive command-line interface, perfectly tailored for media consumption, to a kind of clunky and disjointed GUI. The fact that SageTV still runs in 2024 with media players from 2009/2010 (i.e., the HD200/HD300) really speaks volumes about Jeff's technical vision and his prowess as a software engineer. SageTV still holds a special place in people's hearts.

SageTV made the right decision for themselves and their families to be acquired by Google. Let's face it, the DVR market, especially the hardware-centric ones, is pretty small today, but I do hope the original Sage team knows the impact they had and feel proud of everything they accomplished.

**Comment by Photon0000** [comment]
I'm still not aware of anything better. Sage TV with the SageMC UI is essential for recording and watching from an antenna in a simple easy to use way everybody understands without a learning curve. Nobody has to do anything 99% of the time for the content to be recorded. If I hear about some one off current I check the guide and set it to record or set what is in the guide at that time to record often from multiple channels if it is live broadcast. Record series works even for things that only come around every year or longer and that vary in name because it is the same type of content you watched before so sage records it. Multiple Terabytes of Recordings are automatically organized and quickly browsed. In addition all of my purchased music, DVD and BD content plays from it. All other video types play from it. If any content doesn't work you just do a simple format conversion. It is a single device and UI for everything except streaming and streaming sucks by comparison especially in navigating within content. Streaming that isn't ad supported is expensive and very unsatisfactory for finding content unless you subscribe to a bunch of services. You spend more time looking for content you want to watch than watching content. If you line up stuff to watch in the future a lot of it disappears from the service before you watch. If you want new content on streaming you have to subscribe to something. Everything else is old. With Sage new content is constant from OTA broadcasts. Using the HD homerun app on streaming is no way near as good an experience as SageTV. If you want new movies forget streaming. What you want to watch won't be included in the cost you are already paying and if you pay for a movie and it sucks you regret paying extra for it so you may as well go get it from the library or other source and take advantage of the Highest quality image and latest Audio surround formats plus included extras you can't get at all from streaming. Sage is heavily used. Streaming not much but the live channels will nearly always have something ad supported that you want to watch again. Why would I pay and still have to watch ads? https://www.wired.com/story/amazon-prime-video-ad-free-price-hikes/

**Comment by seansd** [comment]
I only found out this week that the Sage community moved b/c I literally only need to check in once a year or so b/c things just work.  While Roku and streaming has become the main viewing option, we still use Sage often for music, pictures, and for sports watching in linear cases and the occasional recording. It's still by far and away the best option for local media after all these years.

**Reply by Photon0000** [reply]
Typically years go by without my checking the forum also.

I came across some old forum posts I had printed out and just needed to decide if I still needed to keep them for my next build or recycle them, so I looked at the forum and found out it had moved.

I really hope ATSC 3 gets supported for recording. At some point we will be forced to change to ATSC 3 tuners.

**Comment by sircupp** [comment]
I still use it because it just works.  I still have cable card and can record for the time being.  Streaming is getting used more but the ability to access videos for my daughter using the extenders is the main reason.

**Comment by seansd** [comment]
You know. . . given that Sage is mostly Java, I wonder how hard it would be to try to integrate AndroidTV type streaming "channels" (apps).  Having local and streaming would be awesome.  We sort of have this with PlayOn but it's a bit janky.

**Comment by Sagetvmaster** [comment]
This would be awesome, especially if you could trigger apps to open to a specific stream at a specific time, aka sports or big, live events, for automatic recording

**Comment by seansd** [comment]
I mean all the apps work differently but the do all usually have some form of deeplink. . . like on roku they for sure do. . .

**Comment by Sl-a-ck** [comment]
As others have said. Because it works.
(2) HD200's still going strong (touch wood).
+ any client I might wish (PC in Kitchen, PC in WFH office get daily use).
Placeshifter.
Comm skip.
Custom FF and rewind times can be fine tuned.
Recording flexibility.

All of it works without issue.
Why would I mess with that?

Fan art? Nope.
More reliable local streaming of CPU intensive codecs? Yeah that can be a pain. But again, a client box can generally get that done. 
I don't even bother with Kodi anymore. If I'm leaving my comfortable, functional, glitch free realm of SageTV, I am already in windows and will use whatever app I think is best via keyboard and mouse. 9/10 that's just gonna be something lightweight like VLC.

**Comment by way-lo** [comment]
Just one more response here to pile on the love.  I've been using SageTV for probably 15 years now.  Even in the beginning, I found most things intuitive.  Adding on OpenDCT was an undertaking but the dev supplied support was very helpful.

I run 2 clients via the Android app in different rooms of the home and one Windows client on my main PC.  Such a nice feature to pick up any recorded show basically anywhere.

I'm still on Comcast after all this time, and even though they've stopped offering new customers cablecards, they can pry my still functioning CableCard out of my cold, dead hands.  It is a hassle when I renew each year to ensure the new plan will not remove my existing cablecard.

One thing I'd love to see but probably won't happen:  MPV playback with shaders to improve video quality.  I've gotten spoiled with MadVR playback in MPC-HC and dabbled with MPV.  I'll put this on a wishlist if someone knows how to integrate this into a new Windows build.

**Reply by Sagetvmaster** [reply]
Has Comcast really stopped making cable cards available to new customers?  I'm with Cox, but may be moving to a Comcast area, and my four TiVo's won't be happy with that...

**Reply by way-lo** [reply]
The FCC deemed in 9/2020 that Cablecards are no longer required to be supported by cable companies.  It seems like many companies are taking that opportunity to stop offering them to new customers.  Or claiming they don't have any ones to provide customers.

A quick Google search reveals no shortage of Tivo customers posting about cablecard activation issues.

On the bright side, it does looks like Comcast turned back on their cablecard activation site, for what it's worth.  But here's a fairly typical conversation of Comcast users trying hard to get cablecards from Comcast 2 years ago:
https://forums.xfinity.com/conversations/nonx1-service/new-cable-card/62b1eb1126e4b32f2deafa20

**Reply by wayner9** [reply]
Doesn't CableCARD only work on legacy cable? Won't Comcast and other cable companies be phasing that out in the hear future to only support IPTV like Xfinity?

**Reply by way-lo** [reply]
I read somewhere that Comcast will support cablecards until 2031.  Which doesn't necessarily mean they will hand out new ones to new subscribers.  Honestly 2031 sounds way longer than I would have thought.  Once they disappear, how will SageTV be relevant as a DVR product?  Maybe for ATSC?

**Reply by Sagetvmaster** [reply]
I use sage with a regular, set top Cox box. Infrared cable change with USBuirt, and component outs to a Hauppauge HDPVR Gen 1

**Reply by wayner9** [reply]
Wow, that's a long time to keep cable card going.  Here in Canada a bunch of older Sci Atlanta cable boxes are being decommissioned in 8 days on April 30.  Other legacy cable boxes will deep working for a bit longer but they won't allow you to add them to your account.  So I had to switch to Xfinity IPTV - or what Rogers cable calls Ignite but it is a licensed version of Xfinity.

**Reply by emveepee** [reply]
There is also the PowerKEY bug that needs to be considered 
```
Examples of North American cable operators known to have PowerKEY boxes still in the field include Altice USA, Astound Broadband, Breezeline, Comcast, Cox Communications, Charter Communications, Rogers Communications and Videotron.
```

**Reply by wayner9** [reply]
Yep, but apparently there is a workaround.  But Rogers is choosing not to implement it. https://www.lightreading.com/cable-technology/time-warp-could-cause-millions-of-old-powerkey-set-tops-to-go-on-the-blink

**Reply by way-lo** [reply]
@Sagetvmaster Old school!  I remember having a similar setup prior to cable encryption, with the cable coax going directly into a tuner card.  And then moving onto the USBuirt with a low-end video capture card.  And then finally Cablecard.

**Reply by trk2** [reply]
FYI, I have madVR working with Sage although it does not work with recorded content or live tv.  I have a hacked sage.jar that sets the renderer to EVR for live tv or recorded content.  It's works for me but I am not a programmer so I wouldn't want to push this change to github.  I also have Sage transcoding h.265 content to 1080p h.264 when played back oven an extender.

The biggest bonus for madVR is that it allows proper playback of HDR encoded media on PC, but I'm not sure there's enough Sage PC users to make it worthwhile.

**Comment by seansd** [comment]
I would love to hear more about MPV playback @way-lo!  But yeah for me, having a giant library of video and audio content, i have yet to find anything remotely capable of handling that in a living room setting.  Even without TV features, that's still worth it too me

**Reply by way-lo** [reply]
I don't have a good understanding of how video playback is implemented in SageTV, hence my ignorance on if it would be "easy" to add vs require a major overhall.  But I do know MPV is open source and multi-platform with low hardware requirements and is currently used for video playback in other modern apps, most notably Plex.

**Reply by emveepee** [reply]
I added libmpv to my multi-platform client for NextPVR (yauiclient) and it is indeed a nice library to use for h/w desktop clients.  Linux seems better than Windows, which is unusual. Trouble is a lot of the functions are not plug and play (even AC-3 pass through) so it takes tweaking skills.  For watching Dolby Vision  with libplacebo it is pretty neat option on Windows.

However since the HTPC is dead,  I feel it might be better for the limited SageTV development to focus on the mini client alternatives.

**Reply by way-lo** [reply]
Certainly agree that fine tuning MPV for a particular display/GPU is a... shall we call it, "vigorous exercise" for even those users familiar with the nitty gritty behind video playback.

Are you suggesting my on average once a month use of my HTPC/server means it is dead??? =P

**Reply by emveepee** [reply]
A server is not what I consider an HTPC.  For me an HTPC is client a machine in the living or media room, so yes I consider even a good $150-200 NUC a relic, so hide it away as the server.

**Reply by wayner9** [reply]
I think what emveepee means is that the concept of an HTPC has died out.  Microsoft tried with Windows MCE and they stopped years ago.  Several other options have come and gone, like BeyondTV and now SageTV.  Is MythTV even alive these days?

Linear TV is dying so there is less need for the DVR functionality and everything is moving to streaming.  And the newer solutions from cable companies like Xfinity/Ignite are actually very good.  

So a modern cable STB plus a streaming device is in many ways as good as SageTV and much easier to maintain.  HTPCs as a client were always a pain in the butt due to upgrades, drivers, hard drives/SSDs dying, tricky IR support, etc.

**Reply by seansd** [reply]
Yeah but the tinkering is so much fun! And of course I have a huge collection of saved content that none of those cable systems will handle.  I think there's still a need. . .but as you say it needs to pivot towards streaming being the primary input and not linear.  I said this before, I don't think it would be that challenging to make Sage be a home for streaming apps, since all the AndroidTV ones are java based.

**Reply by wayner9** [reply]
Agreed, we like the tinkering because we are geeks, at least I am.  For the archived content you can generally use something like Plex or Jellyfin and they have the advantage of having clients for popular streaming boxes and mobile platforms. And they are generally still being actively maintained.  

I never thought that I would move away from SageTV unless I absolutely had to.  But now I am considering it.

The biggest downside to the cable boxes is that they are $10/month/TV.  I have about ten TVs and they all have SageTV extenders today.  If I could buy the cable boxes for $100-200 it would be an easier decision.  Or if I could access the content from a AppleTV/Roku/ATV, etc. But that isn't available from Rogers, my cable company.

**Reply by way-lo** [reply]
I am holding onto my "HTPC," however one wants to define it, for as long as I can because it serves our family well.  Sure, streaming is the way of the future/now, and can be very convenient, but for SageTV, once it's recorded it's there forever until I make the decision to delete it.  We all keep hearing about how certain TV series, or even outright purchased media, are being removed outside of our control.

I enjoyed the geekiness of building the system and hope to enjoy the fruits for at least several years to come.

My system consists of a SageTV server PC, which also runs my main theater.  The same PC serves as a Plex server.  The theater is mostly for movies (BluRay and streaming) and watching the occasional recorded TV.

In the living room, an Android device serves as a SageTV client, as well as a Plex client.  Serves us well with kids watching lots of Youtube videos.

If I were to take a deep dive into our SageTV usage, certainly it is not nearly as much as it was just several years ago.  The fact many shows appear on Youtube the same day reduces the need.  But having immersive 5.1 audio is important for me, and Youtube doesn't have that for all videos yet.

**Comment by nyplayer** [comment]
> I don't have a good understanding of how video playback is implemented in SageTV, hence my ignorance on if it would be "easy" to add vs require a major overhall. But I do know MPV is open source and multi-platform with low hardware requirements and is currently used for video playback in other modern apps, most notably Plex.

**Comment by Photon0000** [comment]
Been poking around. Seeing the Sage mini client on my Nvidia Shield is likely going away. I don't know what the future holds for SageTV. I'm totally dependent on it. Server and HD300s. Nothing I have tried is a better experience. Everything else is worse user experience with more problems, more clicks, more switching, more time. I'll be with SageTV until the end and I hope that never comes. I am very thankful to those still here, the developers still here, and what has been done to continue the forums and software. SageTV just works, so we aren't here unless we hare making a change to our Sage hardware for some reason as technology evolves.

With Sage there is always something to watch and little to no time spent hunting.

With streaming there is always something OLD to watch quickly and easily because you already saw it. There are often newish clips coming from a live app, not full the content you get from watching the actual TV/app channel live and they are not dated and are typically a few days old so it's old news if you already saw it on TV. Lots of TV episodes from past seasons you already saw. Apps you subscribe to are now getting ads totally ruining the experience. Garbage is being pushed at you. Hard to find anything new to watch and when you spend the time to plan out what you might like, it tends to vanish if you mark it to watch in the future. The quality is extremely variable. Tons of low end content, some mediocre content, and little that you or anyone else rates highly. The platform subscribed to will have a good series of episodes then it's over. Maybe there will be another season next year but usually not. Back to the eternal hunt for something to watch. Watching movies is now history unless you go the the movie theater or subscribe to every movie studios app because the movie you will want to watch is never included with the price you are paying for the apps you subscribe to. It will be on an app you don't subscribe to. So you end up switching apps constantly. After years of this frustrating side environment you give up. I've tried streaming. It sucks. The ads now forced on you when you are paying a subscription for the app content has killed off streaming. SageTV is again all I actually use instead of what I mostly used.

**Comment by ojones-sage** [comment]
Hi All,

Joining this discussion late - as a Verizon FIOS customer -- Sage continues to empower the linear TV experience in my household.  I run two cable-card recording devices (both antiques!) with one serving non-encrypted content via OpenDCT and a second (via a truly ancient w-7 MCE machine) as an intermediary, any encrypted content.  Honestly, nothing beats the WAF that Sage delivers.  It just works.

We use plex as a secondary playback route for mobile users but sage is in charge of recording.  Nothing beats it. Our sage recording machine is virtualized.  Our primary playback device is a NUC (and not a new one),  secondary playback is bare metal on one of our hypervisors (primary job is our Veeam install that has saved my bacon many times but it can hack sage playback too - 2010 vintage server hardware), tertiary playback is via a venerable HD300 that works after the appropriate sacrifice when the moon is in the sky.  When we are on vacation a mini-client on Android TV gets the job done too.

What makes the entire thing stick together though is IP remote control via Control4 and a matrix hdmi switch (for failover when a playback device acts up...).  I owe all of that that to the Sage community. 

I have been a SageTV user since ~2003/2004.  Back then playback was via S-Video...

Hoorah to all who have stuck around.  I know I will.  Go Narflex!
